engine/src/main/java/org/terasology/engine/modes/StateMainMenu.java

```diff
@@ -136,7 +129,9 @@ public void init(GameEngine gameEngine) {
         //guiManager.openWindow("main");
         context.get(NUIManager.class).pushScreen("engine:mainMenuScreen");
         if (!messageOnLoad.isEmpty()) {
-            nuiManager.pushScreen(MessagePopup.ASSET_URI, MessagePopup.class).setMessage("Error", messageOnLoad);
+            TranslationSystem translationSystem = context.get(TranslationSystem.class);
+            MessagePopup popup = nuiManager.pushScreen(MessagePopup.ASSET_URI, MessagePopup.class);
+            popup.setMessage("Error", translationSystem.translate(messageOnLoad, messageOnLoadArguments));
```

Comment:
Is "Error" the title of the popup? Should that be translated, too?

Reply:
Oh, now that you mention it, I suppose it probably is.